frontend/i18n/fr/rules.js

```diff
@@ -0,0 +1,38 @@
+export default {
+  colorRules: {
+    colorRequired: 'La couleur est obligatoire'
+  },
+  labelNameRules: {
+    labelRequied: 'Le nom de l\'étiquette est obligatoire',
```

Comment:
Typo: `Requied` -> `Required`

Reply:
Addressed in https://github.com/doccano/doccano/pull/976/commits/484ce03323a7ded5190dc6c8c425735c0a765b19